js/ad_manager_vast.js

```diff
@@ -776,10 +1114,10 @@ OO.Ads.manager(function(_, $) {
      * The xml is parsed to find any tracking events and then returned as part of an object.
      * @private
      * @method Vast#parseTrackingEvents
-     * @param {array} tracking to add the tracking info to and return.
-     * @param {xml} xml The data of the ad with tracking info.
-     * @param {array} trackingEvents List of events that are tracked, if null then it uses the global one.
-     * @returns {array} tracking An array of tracking items.
+     * @param {array} tracking to add the tracking info to and return
+     * @param {xml} xml The data of the ad with tracking info
+     * @param {array} trackingEvents List of events that are tracked, if null then it uses the global one
+     * @returns {array} An array of tracking items.
```

Comment:
i thought "array" is not a supported type by our JSDoc interpreter?  I thought it had to be object?